admin/acceptance_tests/interface_test.py

```diff
@@ -0,0 +1,74 @@
+# pylint: disable=no-self-use
+
+import pytest
+
+from . import AbstractViewsTests
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures('dbsession')
+def interface_test_data(dbsession):
+    from c2cgeoportal_commons.models.main import Interface
+
+    dbsession.begin_nested()
+
+    interfaces = []
+    for i in range(0, 25):
+        interface = Interface(name='interface_{}'.format(i),
+                              description='description_{}'.format(i))
+
+        dbsession.add(interface)
+        interfaces.append(interface)
+
+    yield {
+        'interfaces': interfaces
+    }
+    dbsession.rollback()
+
+
+@pytest.mark.usefixtures('interface_test_data', 'transact', 'test_app')
+class TestInterface(AbstractViewsTests):
+
+    _prefix = '/interfaces'
+
+    def test_index_rendering(self, test_app):
+        resp = self.get(test_app)
+
+        self.check_left_menu(resp, 'Interfaces')
+
+        expected = [('_id_', '', 'false'),
+                    ('name', 'Name', 'true'),
+                    ('description', 'Description', 'true'),
+                    ('layers', 'Layers', 'false'),
+                    ('theme', 'Themes', 'false')]
+        self.check_grid_headers(resp, expected)
+
+    def test_grid_complex_column_val(self, test_app, interface_test_data):
+        first_row = test_app.post(
+            '/interfaces/grid.json',
+            params={
+                'current': 1,
+                'rowCount': 10
+            },
+            status=200
+        ).json['rows'][0]
+
+        first_interface = interface_test_data['interfaces'][0]
+
+        assert first_interface.id == int(first_row['_id_'])
+        assert first_interface.name == first_row['name']
+        assert first_interface.description == first_row['description']
```

Comment:
I would like to see tests on complex columns (columns with custom renderer like themes, layers)